lib/parser.js

```diff
@@ -60,11 +60,13 @@ class Parser extends EventEmitter {
       entry.value = entry.value[entry.name]
 
       if (property.key.name === 'computed') {
-        /* istanbul ignore else */
         if (entry.value instanceof utils.NodeFunction) {
           entry.dependencies = utils.getDependencies(entry.value, this.source.script)
         } else if (entry.value instanceof Object && entry.value.get instanceof utils.NodeFunction) {
           entry.dependencies = utils.getDependencies(entry.value.get, this.source.script)
+        } else {
+          const error = new Error('Computed property must be a function or an object with a get() function')
```

Comment:
I think the parser should not emit Vue syntax error here. Syntax error must be throwed by #13
The parser should only parse the `get` function and ignore the others one.

Reply:
I added this case because otherwise the coverage will go down (slightly) as there is no "else" case. It is fine by me to remove the else block here, should I do that?

Reply:
Yes, the `else` branch is not required in this case.